### PR TITLE
Fix on running locally

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cciutils/__init__.py [hunk 1]
```diff
@@ -8,6 +8,21 @@
...
+
+    remote_lines = subprocess.check_output(["git", "remote", "--verbose"]).decode().split("\n")
+    remote_match = (
+        re.match(r".*git@github.com:(.*).git .*", remote_lines[0]) if len(remote_lines) >= 1 else None
+    )
+
+    if remote_match:
+        return remote_match.group(1)
+
+    return "camptocamp/project"
```
Comment (arnaud-morvan) [inline c2cciutils/__init__.py:23]: May log a warning in case there is no match.

**Review by arnaud-morvan — APPROVED** [review]
Looks good, +1 to be able to run publish locally.
In case we have a blocking issue and CI does not pass this can be really useful.

File: c2cciutils/__init__.py (hunk 1 above)
Comment (sbrunner) [reply c2cciutils/__init__.py:25]: +1 Done :-)

File: c2cciutils/__init__.py [hunk 2]
```diff
@@ -8,6 +8,23 @@
...
+
+    remote_lines = subprocess.check_output(["git", "remote", "--verbose"]).decode().split("\n")
+    remote_match = (
+        re.match(r".*git@github.com:(.*).git .*", remote_lines[0]) if len(remote_lines) >= 1 else None
+    )
+
+    if remote_match:
+        return remote_match.group(1)
+
+    print("WARNING: the GitHub reposotory isn't found, using 'camptocamp/project'")
```
Comment (arnaud-morvan) [inline c2cciutils/__init__.py:23]:
```suggestion
    print("WARNING: the GitHub repository isn't found, using 'camptocamp/project'")
```

File: c2cciutils/__init__.py [hunk 3]
```diff
@@ -8,6 +8,23 @@
...
+
+    return "camptocamp/project"
+
+
 def merge(default_config, config):
     """
     Deep merge the dictionaries on dicktionaries only (not on array).
```
Comment (arnaud-morvan) [inline c2cciutils/__init__.py:30]:
```suggestion
    Deep merge the dictionaries (on dictionaries only, not on array).
```

**Pull request merged**